app/slashcommands/command_custom_status.go

```diff
@@ -60,18 +59,10 @@ func (*CustomStatusProvider) DoCommand(a *app.App, args *model.CommandArgs, mess
 		}
 	}
 
-	user, err := a.GetUser(args.UserId)
-	if err != nil {
-		mlog.Error(err.Error())
-		return &model.CommandResponse{Text: args.T("api.command_custom_status.app_error"), ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
-	}
-
-	timezoneEnabled := *a.Config().DisplaySettings.ExperimentalTimezone
 	customStatus := &model.CustomStatus{
-		Emoji:     DefaultCustomStatusEmoji,
-		Text:      message,
-		Duration:  DefaultCustomStatusDuration,
-		ExpiresAt: calculateExpiryTime(user.Timezone, timezoneEnabled),
+		Emoji:    DefaultCustomStatusEmoji,
+		Text:     message,
+		Duration: DefaultCustomStatusDuration,
```

Comment:
use the constant defined in `model` package
```suggestion
		Duration: model.DefaultCustomStatusDuration,
```